database/src/main/scala/cromwell/database/sql/tables/JobStoreResultSimpletonEntry.scala

```diff
@@ -0,0 +1,11 @@
+package cromwell.database.sql.tables
+
+case class JobStoreResultSimpletonEntry
+(
+  simpletonKey: String,
+  simpletonValue: String,
+  wdlType: String,
+  jobStoreEntryId: Int,
```

Comment:
The column is `JOB_STORE_ID` in the database, so the field name should match, aka `jobStoreId`.